README.md

```diff
@@ -302,6 +302,9 @@ description: # can be multiline, see syntax here https://yaml-multiline.info/
 # or category ID (e.g. 161/27)
 category: Notebooks
 
+prefix: # optional prefix to be added to the description overriding the default prefix
```

Comment:
I would prefer `description_prefix` and `description_suffix` to be clear this relates to the description and not e.g. the title. WDYT?

Reply:
I am sorry if I probably expained it poorly, I meant either leave it as is:

```yaml
ad_defaults:
  description:
    prefix: ""
    suffix: ""
```

or change it to:

```yaml
ad_defaults:
  description_prefix: ""
  description_suffix: ""
```

but not:

```yaml
ad_defaults:
  description:
    description_prefix: ""
    description_suffix: ""
```